flybuy.js

```diff
@@ -301,6 +314,7 @@ class Flybuy {
     if (this.provider === 'google') {
       let position = {lat, lng};
       let marker = new google.maps.Marker({map: map, position: position, draggable: false});
+      marker
```

Comment:
is this necessary?

Reply:
No, that was a really stupid mistake that got checked in. I'm working on Mapbox support right now and fixed it in that branch.